src/connectedvmware/README.rst

```diff
@@ -0,0 +1,5 @@
+Microsoft Azure CLI 'connectedvmware' Extension
+==========================================
+
+This package is for the 'connectedvmware' extension.
+i.e. 'az connectedvmware'
```

Comment:
You can add some commands usage in readme file, refer to https://github.com/Azure/azure-cli-extensions/tree/master/src/vmware